crates/rolldown/src/ast_scanner/impl_visit.rs

```diff
@@ -184,7 +192,17 @@ impl<'me, 'ast> Visit<'ast> for AstScanner<'me> {
           if request.span().is_empty() {
             ImportRecordMeta::IS_UNSPANNED_IMPORT
           } else {
-            ImportRecordMeta::empty()
+            let mut meta = ImportRecordMeta::empty();
+            if self
+              .nearest_parent_stmt_info
```

Comment:
```suggestion
              .nearst_ancestor_stmt_node_info
```

It seems that the `nearest_parent_stmt_info` is actually the `stmt` that contains this `expr` node. It might be not accurate to said that the `nearest_parent_stmt_info` is the parent statement of  the `expr` node.

Maybe `nearst_ancestor_stmt_node_info?`